pkg/master/reconcilers/mastercount.go

```diff
@@ -93,7 +95,7 @@ func (r *masterCountEndpointReconciler) ReconcileEndpoints(serviceName string, i
 	if !formatCorrect {
```

Comment:
does checkEndpointSubsetFormat detect when the nodeName associated with an IP is incorrect/stale? if so, how does this avoid dueling with 1.14 API servers? if not, how does this ensure the endpoint gets updated if the nodeName changes?

Reply:
I have added validation for `mastercount` reconciler. for `lease` reconciler nodename updates on every reconcile iteration while updating a TTL for our key.